docker-compose.override.yml

```diff
@@ -0,0 +1,41 @@
+version: '3'
+services:
+  express-server:
+    image: ${DOCKERHUB_NAME}/${IMAGE_NAME}:${IMAGE_VERSION}
```

Comment:
Don't need `image` because it's in `docker-compose.yml` and this file will inherit that setting.